install-dependencies_lducas.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#git clone -b fpylll-changes https://github.com/lducas/fplll
+cd fplll
+./autogen.sh
+if test "$1" != ""; then
```

Comment:
Hum, I committed that after the pull request I thought. Indeed, this is just for my own setup. Not sure how to de-include it from the pull-request